debian/cloud-init.postinst

```diff
@@ -309,7 +309,7 @@ fix_lp1889555() {
     [ -f /var/lib/cloud/instance/sem/config_grub_dpkg ] || return 0
 
     # Don't do anything unless we have grub
-    [ -x /usr/sbin/grub-install ] || return 0
+    [ -x "$(which grub-install)" ] || return 0
```

Comment:
Let's align with how we are doing this in general to check for presence of a command in your path in ds-identify and other postinst script functions
```suggestion
    command -v grub-install > /dev/null 2>&1 || return 0
```

Reply:
+1 on this: `which` is half deprecated (prints a deprecation warning in Debian, which was silenced in Ubuntu *for now*).